util/pyclient/test_msgs.py

```diff
@@ -32,8 +33,9 @@ def test_unpack_request(self):
         client_id = 4
         req_seq_num = 1
         read_only = True
-        packed = bft_msgs.pack_request(client_id, req_seq_num, read_only, msg)
-        (header, unpacked_msg) = bft_msgs.unpack_request(packed)
+        cid = str(req_seq_num)
+        packed = bft_msgs.pack_request(client_id, req_seq_num, read_only, cid, msg)
+        (header, unpacked_msg) = bft_msgs.unpack_request(packed, len(cid))
```

Comment:
Can you add an `assertEqual` to check that the cid is what you expect please?

Reply:
Fixed